gui/src/components/commons/Radio.tsx

```diff
@@ -35,7 +35,10 @@ export function Radio({
         >
           <input
             type="radio"
-            className="text-accent-background-30 focus:ring-transparent focus:ring-offset-transparent focus:outline-transparent"
+            className={classNames(
```

Comment:
Please use multiline strings (with backticks), classnames has to run which might result in flicker.